.github/workflows/generators.yml

```diff
@@ -0,0 +1,30 @@
+name: ArtifactHub Component Generation
```

Comment:
Considering other repos in the future, should this be a generic title?

Reply:
Changed to `Static Component Generation`